### ISSUE TITLE
Resource template required files

### ISSUE THREAD

**Issue by allanaaa** [omeka/omeka-s#2041]
I have a resource template with several required fields. 
When I apply the template to an existing item, then try to save, I get an error about the first required field that stops me from moving on. I *don't* get an error about the other field(s), in this case the creator field. 
When I put in a description and save, it will save the new description value, give me an error about the creator, and fail to save the template change. 

![template-required](https://user-images.githubusercontent.com/28012148/227991133-3ba9e6e9-ad3a-49b3-993a-4bb78558610d.PNG)
![template-required2](https://user-images.githubusercontent.com/28012148/227991142-cacda016-b03e-42de-b2dd-9362a7e7ed2f.PNG)

What I'd like instead is for the creator (and any other required fields) error to work the same way as the description error does, and stop me from moving on until I fill everything out. I presume this is "We only show one error, the first we come across, and then don't do any more checks after that" rather than "We only check a few particular properties when they are required", which would be odd.

**Comment by jimsafley** [comment]
This is due to an errant "value" class in the URI data type's markup ([here](https://github.com/omeka/omeka-s/blob/v4.0.1/application/view/common/data-type/uri.phtml#L14)). We need to remove the class, but it affects the styling.

@kimisgold Would you take a look at this? We need to replace that "value" class.

**Comment by kimisgold** [comment]
Edited the class to be more specific to the input. It's ready to test @allanaaa

**Comment by allanaaa** [comment]
It's giving an error for all required fields now. But it's pulling both the original field name and the alternate titles in an unwieldy way:

![creatorauthor](https://user-images.githubusercontent.com/28012148/228288273-0c5b17d0-c92e-4ebc-aff7-5775dc2a0454.PNG)

**Comment by zerocrates** [comment]
The code attempts to do what I think is expected here: it just uses the label. The problem is that the non-custom label is there in the markup, hidden, so the code here is finding both the original and the alternate label. Something simple like calling `.last()` before getting the text may be sufficient.

**Comment by jimsafley** [comment]
You should now only get one label, @allanaaa

**Comment by allanaaa** [comment]
No luck on this yet, sorry.

![restemp](https://user-images.githubusercontent.com/28012148/229875430-3f211b21-a2c1-4e1b-aa70-1000e8156f41.PNG)

**Comment by zerocrates** [comment]
Just to make sure, do the whole "clear cache" thing: this was a Javascript-only fix so if you're looking at the old Javascript you'd still see it.

**Comment by zerocrates** [comment]
The basic deal is, value markup needs to have a `to-require` class on whatever needs to be filled in for the value to be considered complete. Probably for these ones that aren't triggering, they don't have a to-require on anything.

Both are a little oddball in that they involve having several inputs each of which is optional. ValueSuggest may be able to get by marking the main visible input as to-require but I suspect something more complex may be required for NumericDataTypes (or maybe just requiring the "first" input will be sufficient there).

**Comment by allanaaa** [comment]
Yep, it's fields with Value Suggest or some other data type assigned that don't always (but sometimes do) catch an error before you save. Instead you see this after saving. 

![required](https://github.com/omeka/omeka-s/assets/28012148/0f576f75-9f95-40b5-af04-a63e805fd351)

This means that, if you enter in information elsewhere (like adding map markers on the Map tab), you'll lose that work when you save the item. 

It looks like it'll report the first one it comes across each time, instead of all the ones that need to be fixed:
![one-error](https://github.com/omeka/omeka-s/assets/28012148/75d33ae1-8daf-476a-ab93-d24891cc44c5)
